src/items/menu.js

```diff
@@ -138,7 +138,8 @@ export default class MenuPanelItem extends PanelItem {
           } else if (action === 'logOut') {
             logout(false);
           }
-        }
+        },
+        toggle: true
```

Comment:
That's the only change left in this PR now 😬

Reply:
Noice! :)